data/brands/shop/money_lender.json

```diff
@@ -114,6 +114,19 @@
         "shop": "money_lender"
       }
     },
+    {
+      "displayName": "Crefisa",
+      "id": "crefisa-46cd18",
+      "locationSet": {"include": ["br"]},
+      "tags": {
+        "brand": "Crefisa",
+        "brand:website": "https://www.crefisa.com.br/",
+        "brand:wikidata": "Q10261396",
+        "brand:wikipedia": "pt:Crefisa",
+        "phone": "+55 86 3221 2476",
```

Comment:
Just an FYI, we also tend not to include contact details in this project, so you can also remove the phone number 👍

Reply:
Okay, I have remove entry "phone" from the file "money_lender.json".